pkg/client/src/app/api/models.tsx

```diff
@@ -347,6 +347,7 @@ export interface TaskData {
     binary: boolean;
     withDeps: boolean;
     artifact: string;
+    diva: boolean;
```

Comment:
This looks correct. Ref: https://github.com/konveyor/tackle2-addon-windup/pull/42